src/components/MapContainer/olmin.css

```diff
@@ -0,0 +1,267 @@
+.ol-box {
```

Comment:
@ivanoats added this files to include css styles necessary with openlayers, we can also define our own components, but this is standard stylesheet provided by https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/css/ol.css